src/main/java/org/frc5687/rapidreact/commands/catapult/IdleCatapult.java

```diff
@@ -23,14 +23,8 @@ public void initialize() {
     @Override
     public void execute() {
         super.execute();
-        double speed = _oi.getWinchMotorSpeed();
-        metric("winch speed", speed);
         _catapult.setWinchMotorSpeed(0);
-//        double speedSpring = _oi.getSpringMotorSpeed();
-//        metric("spring speed", speedSpring);
-//        _catapult.setSpringMotorSpeed(0.0);
         _catapult.setSpringMotorSpeed(_oi.getSpringMotorSpeed());
```

Comment:
Do you still need manual spring motor control?

Reply:
oh, shoot. I thought that was holding position. So no.